src/svymean.jl

```diff
@@ -102,23 +102,39 @@ end
 
 """
 Inner method for `svyby` for StratifiedSample
+Calculates domain mean and its std error, based on pg392 Sarndal.
 """
-function svymean(x::AbstractVector, sampfraction::AbstractVector,strata::AbstractVector,design::StratifiedSample)
-    df = DataFrame(x = x, sampfraction = sampfraction,strata = strata)
+function svymean(x::AbstractVector, popsize::AbstractVector,sampsize::AbstractVector,sampfraction::AbstractVector,strata::AbstractVector)
+    df = DataFrame(x = x, popsize = popsize, sampsize = sampsize, sampfraction = sampfraction,strata = strata)
     nsdh = []
     substrata_domain_totals = []
-    sampfractions = []
+    Nh = []
```

Comment:
We should try to have more readable variable names, or at least comments describing each variable. This would be the population size corresponding to each stratum, for example